src/python/pants/java/util.py

```diff
@@ -274,7 +275,8 @@ def safe_classpath(classpath, synthetic_jar_dir, custom_name=None):
   else:
     synthetic_jar_dir = safe_mkdtemp()
 
-  bundled_classpath = relativize_classpath(classpath, synthetic_jar_dir)
+  # Quote the paths so that if they contain reserved characters can be safely passed to JVM classloader.
+  bundled_classpath = map(lambda x: urllib.quote(x), relativize_classpath(classpath, synthetic_jar_dir))
```

Comment:
I think you can replace `lambda x: urllib.quote(x)` with just `urllib.quote`, no?

Reply:
yes indeed, good catch!